ngrok/examples/axum.rs

```diff
@@ -42,10 +42,12 @@ async fn start_tunnel() -> anyhow::Result<HttpTunnel> {
         .await?
         .http_endpoint()
         // .allow_cidr("0.0.0.0/0")
+        // .app_protocol("http2")
```

Comment:
We intentionally left this out for this example, and want to link the context:
https://github.com/ngrok/ngrok-rust/pull/130#discussion_r1428131186